docs/sources/github.md

```diff
@@ -11,34 +11,43 @@ Follow the following steps:
 
 1. From your organization, register a [GitHub App](https://docs.github.com/en/apps/creating-github-apps/registering-a-github-app/registering-a-github-app#registering-a-github-app)
    with following permissions with **Read Only**:
-   - Repository:
-     - Contents: for reading commits and comments
-     - Issues: for listing issues, comments, assignees, etc.
-     - Metadata: for listing repositories and branches
-     - Pull requests: for listing pull requests, reviews, comments and commits
-   - Organization
-     - Administration: for listing events from audit log
-     - Members: for listing teams and their members
+    - Repository:
+        - Contents: for reading commits and comments
+        - Issues: for listing issues, comments, assignees, etc.
+        - Metadata: for listing repositories and branches
+        - Pull requests: for listing pull requests, reviews, comments and commits
+    - Organization
+        - Administration: for listing events from audit log
+        - Members: for listing teams and their members
 
 NOTES:
 - We assume that ALL the repositories are going to be list **should be owned by the organization, not the users**.
 - Enterprise Cloud is required for this connector.
 
 Apart from Github instructions please review the following:
 - "Homepage URL" can be anything, not required in this flow but required by Github.
-- Callback URL is required for next step, it can be any URL (http://localhost, for example);
 - Webhooks check can be disabled as this connector is not using them
 - Keep `Expire user authorization tokens` enabled, as GitHub documentation recommends
-2. Once is created please generate a new client secret. We will both (clientId and clientSecret) in next steps.
-3. Install the application in your organization.
+2. Once is created please generate a new `Private Key`.
+3. It is required to convert the format of the certificate downloaded from PKCS#1 in previous step to PKCS#8. Please run following command:
+```shell
+openssl pkcs8 -topk8 -inform PEM -outform PEM -in {YOUR DOWNLOADED CERTIFICATE FILE} -out priv8.pem -nocrypt
```

Comment:
I'd rename `priv8.pem` to something more meaningful, like `gh_pk_pkcs8.pem`. (here and in L50) 

And also I'd suggest a step 7 to remove the downloaded files pkcs1 and pkcs8 and store the PK somewhere else if they want to (not sure if the PK can be downloaded at will or only once)